src/runtime.js

```diff
@@ -196,6 +196,11 @@ var Runtime = {
 #if ASSERTIONS
       assert(args.length == sig.length-1);
 #endif
+      if (Module['usingWasm']) {
+        // look up function by name and call it directly
+        name = Module['indirectFunctions'][ptr]
+        return Module[name].apply(null, args);
```

Comment:
Yes, much faster. When it's a static string, the lookup can be optimized by the JIT, and often eliminated/hardcoded (unless the object's "hidden class/shape" changes, which the JIT must guard on).